tests/regression/nextcloud-rule-exclusions-plugin/9508720.yaml

```
@@ -0,0 +1,46 @@
+              as one of the first pumpkin soup recipes we ever published online. Since then, it’s become one of the most popular creamy pumpkin soup recipes in Australia, with more than 350 rave reviews to date.\n\n
+              It’s creamy without being heavy, delicately spiced without being spicy, and hearty without being too rich. Pretty much everything we look for in a pumpkin soup (and we’ve tried
+              a lot of pumpkin soup recipes over the years!).\n\nServe this pumpkin soup with a satisfyingly crusty piece of bread. If you like, you can sprinkle your soup with chilli flakes (or cayenne pepper)
+              or a pinch of smoked paprika. \n\nIf you're not sure how best to cut and prep your pumpkin (be it butternut squash or another type of pumpkin) for your soup, check out our video guide at the bottom
+              of the method steps.\n\nWe also have a flavourful vegan pumpkin soup recipe for those who prefer a dairy-free soup. Looking for more easy soup recipe ideas? Try our 15-minute chicken and corn noodle
+              soup. You could also make pumpkin soup in a slow cooker.","url":"https://www.taste.com.au/recipes/pumpkin-soup/e54ab2b5-7d19-4e4b-b383-781292dc4bfb?r=quickeasy/biccuul7",
+              "image":"/best-easy-pumpkin-soup-recipe-185570-1.jpg","prepTime":"PT00H10M00S","cookTime":"PT00H40M00S","totalTime":"PT00H50M00S","recipeCategory":["Soup","Easy"],
+              "keywords":"Pumpkin,Soup,Easy","recipeYield":6,"tool":[],"recipeIngredient":["","1L Massel chicken style liquid stock or vegetable liquid stock","1 large potato, peeled, diced",
+              "1kg peeled pumpkin, diced","1/2 tsp freshly grated nutmeg","1 tsp ground cumin","1/2 tsp ground coriander","1 garlic clove, crushed","1 leek, white part only, finely sliced","1 onion, finely chopped",
+              "2 tbsp Coles Classic Olive Oil"],"recipeInstructions":["Heat oil in a large saucepan over low heat, add onion and leek and cook for 2-3 minutes, until softened but not coloured.\n",
+              "Add garlic, coriander, cumin, and nutmeg and cook, stirring, for 30 seconds. Add pumpkin, potato and stock and bring to the boil. Turn heat to low, cover and simmer for 30 minutes.
+              Allow to cool slightly, then blend in batches.\n","Return soup to pan, stir through cream and reheat gently. Season and add a little more nutmeg if desired.\n"],
+              "nutrition":{"calories":"206cal","proteinContent":"3.1g","fatContent":"12.9g","saturatedFatContent":"5.0g","carbohydrateContent":"21.9g","sugarContent":"5.8g","sodiumContent":"563.8mg"},
+              "valueInit":{"id":0,"name":"","description":"","url":"","image":"","prepTime":"","cookTime":"","totalTime":"","recipeCategory":"","keywords":"","recipeYield":"","tool":[],"recipeIngredient":[],
+              "recipeInstructions":[],"nutrition":{}}}
+            version: HTTP/1.1
+          output:
+            no_log_contains: |
```

Careful here, without the chomping indicator, the regular expression will contain a line break at the end.
```suggestion
            no_log_contains: |-
```